### PR TITLE
Create CCC-TAC-Minutes-2022-10-06.md

### PR DESCRIPTION
**Pull request by krtaylor**

### CODE REVIEW

File: TAC/Meetings/2022/2022-10-06/CCC-TAC-Minutes-2022-10-06.md [hunk 1]
```diff
@@ -0,0 +1,149 @@
...
+   * Naveen Cherukuri (NVIDIA)
+   * Nick Vidal (Profian)
+   * Penglin Yang (China Mobile)
+   * Xinxin Fan (IoTex)
+   * Mark Novak (JP Morgan Chase)
+   * Ahmad Atamli (NVIDIA)
+   * Alec Fernandez (Microsoft)
+   * Andy Dellow (Huawei UK)
+   * Francois-Xavier Merseille (Thales)
+   * Henk Birkholz (IETF)
```
Comment (thomas-fossati) [inline TAC/Meetings/2022/2022-10-06/CCC-TAC-Minutes-2022-10-06.md:70]:
```suggestion
   * Henk Birkholz (Fraunhofer SIT)
```

File: TAC/Meetings/2022/2022-10-06/CCC-TAC-Minutes-2022-10-06.md [hunk 2]
```diff
@@ -0,0 +1,149 @@
...
+
+* DT reviewed the current open PRs for terminology
+	* Remove confidential container term #141
+	  * https://github.com/confidential-computing/governance/pull/141
+	  * Closed, instead use #142
+	* Fix confidential container definition #142
+	  * https://github.com/confidential-computing/governance/pull/142
+	  * Will merge by next meeting if no objections
+	* Proposal to include attestation in definition of Confidential Computing #135
+	  https://github.com/confidential-computing/governance/pull/135
```
Comment (dcmiddle) [inline TAC/Meetings/2022/2022-10-06/CCC-TAC-Minutes-2022-10-06.md:131]: Do we need to capture an explicit TAC vote on this (c.f. board decision on this verbiage 2022/10/19 based on communicated TAC vote).

File: TAC/Meetings/2022/2022-10-06/CCC-TAC-Minutes-2022-10-06.md [hunk 3]
```diff
@@ -0,0 +1,149 @@
...
+   * Microsoft - Dave Thaler (TAC chair) +
+   * Red Hat - Lily Sturmann  / Dimitrios Pendarakis
+
+   " + " indicates attending
+
+***Other Attendees***
+
+   * Alec Fernandez (Microsoft)
+   * Eric Voit (Cisco)
+   * Francois-Xavier Marseille (THALES)
```
Comment (dthaler) [inline TAC/Meetings/2022/2022-10-06/CCC-TAC-Minutes-2022-10-06.md:58]:
```suggestion
   * Francois-Xavier Marseille (Thales)
```
for consistency with line 69

File: TAC/Meetings/2022/2022-10-06/CCC-TAC-Minutes-2022-10-06.md [hunk 4]
```diff
@@ -0,0 +1,149 @@
...
+   * Linux Plumbers discussion on AP-TEE - https://lpc.events/event/16/contributions/1167/attachments/1069/2071/LPC2022_Sahita_Conf_Comp_on_RISCV.pdf
+   * Salus (TSM) Github - https://github.com/rivosinc/salus
+   * Platform Security Model Specification Github - https://github.com/riscv-non-isa/riscv-platform-security-model
+* ACTION: [Lily] recommendation is to make sure Keystone is participating
+
+**Meeting Minute Approval**
+
+Proposed: That the minutes of the September 22, 2022 meeting of the Technical Advisory Council of the Confidential Computing Consortium as distributed to the members of the TAC in advance of this meeting are hereby adopted and approved.
+
+   * Quorum was reached and the minutes were approved by the voting members of the TAC. KT to amend if there was a missing action or recommendation from the MPC presentation.
```
Comment (dthaler) [inline TAC/Meetings/2022/2022-10-06/CCC-TAC-Minutes-2022-10-06.md:94]: Was there?
Maybe an action item related to Veracruz?

**Comment by dthaler** [comment]
@krtaylor can you respond to the comments in github above?

File: TAC/Meetings/2022/2022-10-06/CCC-TAC-Minutes-2022-10-06.md (hunk 4 above)
Comment (AlecFernandez) [reply TAC/Meetings/2022/2022-10-06/CCC-TAC-Minutes-2022-10-06.md:94]: Action item related to Veracruz arose during the 9/22 meeting, Thomas Fossati took an action to reach out to Jordan Brandt of MPC alliance given the large overlap in their respective charters.  Here is the link to the exchange: https://youtu.be/IoUOZZrP_-A?t=4336

Comment (thomas-fossati) [reply TAC/Meetings/2022/2022-10-06/CCC-TAC-Minutes-2022-10-06.md:94]: hi @thomas-fossati the action holder here :-)

I think I said it during last meeting that I had put Derek, Hugo and Jordan in contact around the 23rd of September.  I will check again if there's been any progress on that front, but apart from that the action should be complete.

Comment (krtaylor) [reply TAC/Meetings/2022/2022-10-06/CCC-TAC-Minutes-2022-10-06.md:94]: I waited to commit the previous minutes to amend with the changes Alec suggested

File: TAC/Meetings/2022/2022-10-06/CCC-TAC-Minutes-2022-10-06.md (hunk 2 above)
Comment (dthaler) [reply TAC/Meetings/2022/2022-10-06/CCC-TAC-Minutes-2022-10-06.md:131]: "The TAC voted to approve the change and take it to the board for ratification."

**Pull request merged**